.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: "ci"
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  Spec:
+    uses: "puppetlabs/cat-github-actions/.github/workflows/module_ci.yml@main"
+    secrets: "inherit"
+
+  Acceptance:
+    uses: "puppetlabs/cat-github-actions/.github/workflows/module_acceptance.yml@main"
```

Comment:
just curious do we need @main or does it know the latest?

Reply:
we pull off the main branch just